executor/foreign_key.go

```diff
@@ -187,6 +219,10 @@ func (fkc *FKCheckExec) doCheck(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	if fkc.stats != nil {
+		fkc.stats.Check = time.Since(start)
+		fkc.stats.Total = fkc.stats.Check
```

Comment:
We just left the update of the `Total` to the defer func?

Reply:
done.